src/rabbit_control_misc.erl

```diff
@@ -69,25 +69,27 @@ step_with_exit_handler(AggregatorPid, Ref, Fun, Item) ->
             ok
     end.
 
-wait_for_info_messages(Pid, Ref, ArgAtoms, DisplayFun, Timeout) ->
-    _ = notify_if_timeout(Pid, Ref, Timeout),
-    wait_for_info_messages(Ref, ArgAtoms, DisplayFun).
+wait_for_info_messages(Pid, Ref, DisplayFun, Timeout, Chunks) ->
```

Comment:
I'd rename `Chunks` to `NodesCount` and `NodesLeft` or something like that, so that it is obvious what exactly the function is waiting for.